src/material/form-field/form-field.md

```diff
@@ -16,36 +16,52 @@ The following Angular Material components are designed to work inside a `<mat-fo
 
 ### Form field appearance variants
 The `mat-form-field` supports 4 different appearance variants which can be set via the `appearance`
-input. The `legacy` appearance is the default style that the `mat-form-field` has traditionally had.
-It shows the input box with an underline underneath it. The `standard` appearance is a slightly
-updated version of the `legacy` appearance that has spacing that is more consistent with the `fill`
-and `outline` appearances. The `fill` appearance displays the form field with a filled background
-box in addition to the underline. Finally the `outline` appearance shows the form field with a
-border all the way around, not just an underline.
-
-There are a couple differences to be aware of between the `legacy` appearance and the newer
-`standard`, `fill`, and `outline` appearances. The `matPrefix` and `matSuffix` elements are center
-aligned by default for the newer appearances. The Material Design spec shows this as being the
-standard way to align prefix and suffix icons in the newer appearance variants. We do not recommend
-using text prefix and suffixes in the `fill` and `outline` variants because the label and input do not have the same
-alignment. It is therefore impossible to align the prefix or suffix in a way that looks good when
-compared with both the label and input text.
-
-The second important difference is that the `standard`, `fill`, and `outline` appearances do not
+input. The `legacy` and `standard` appearances implement an older version of the spec where
```

Comment:
Nit: The order seems off as someone learning about the appearances. It seems odd to start with the two appearances that don't match the spec. I'd expect these to be mentioned near the bottom in a section about accepted but not supported appearances.

Then again, we default to `legacy` so *shrug*. Would be cool to change the default (and probably transform the code into Google to keep supporting legacy as default)

Reply:
Yeah that makes sense, I just left it in the order I found it, but it would probably be good to switch it